### PR TITLE
Add new prop 'additionalRoutingParams' to pass further parameters to the routing request.

### PR DESCRIPTION
**Pull request by edellucien**
Add new prop 'additionalRoutingParams' to pass further parameters to the routing request.

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [ ] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] The title means something for a human being.
- [x] The title contains [WIP] if it's necessary.
- [x] Labels applied. if it's a release? a hotfix?
- [x] The new class' members & methods are well documented
- [ ] Tests added.

### CODE REVIEW

File: src/ol/controls/RoutingControl.js [hunk 1]
```diff
@@ -65,6 +66,9 @@ class RoutingControl extends Control {
     /** @ignore */
     this.mot = options.mot || 'bus';
 
+    /** @ignore */
+    this.additionalRoutingParams = options.additionalRoutingParams || {};
```
Comment (oterral) [inline src/ol/controls/RoutingControl.js:70]: call it routingApiParams

Comment (edellucien) [reply src/ol/controls/RoutingControl.js]: Alright. the prop is renamed

Comment (mhaertwig) [reply src/ol/controls/RoutingControl.js]: What is the strategy? When do we expose parameters via JavaScript and when do we use the additional parameters?

Comment (mhaertwig) [reply src/ol/controls/RoutingControl.js]: ... forget it. The above parameters are just defaults. So only `mot` and `via` are exposed.
For me it's fine like this.

**Review by oterral — APPROVED** [review]
lgtm

**Pull request merged**